test/unit/options.spec.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+describe('options', function () {
+  describe('#defaultOptionsPath', function () {
+    var options = require('../../lib/options');
+
+    it('property should have value \'test/mocha.opts\'', function () {
+      expect(options.defaultOptionsPath).to.equal('test/mocha.opts');
+    });
+
+    it('property should NOT be writable/configurable', function () {
+      expect(function () {
+        options.defaultOptionsPath = '/new/path/mocha.opts';
+      }).to.throwError(function (error) {
+        expect(error).to.be.a(TypeError);
+      });
+    });
+  });
+
+  describe('#path', function () {
+    var options = require('../../lib/options');
+
+    it('should return user passed options path if it exists, or the default path', function () {
+      var argv, path;
+
+      argv = [];
+      path = options.path(argv, options.defaultOptionsPath);
+      expect(path).to.equal(options.defaultOptionsPath);
+
+      argv = ['--opts', '/user/passed/mocha.opts'];
+      path = options.path(argv, options.defaultOptionsPath);
+      expect(path).to.equal('/user/passed/mocha.opts');
+    });
+  });
+
+  describe('#read', function () {
+    var options = require('../../lib/options');
+
+    it('throws when given a path that does not exist', function () {
+      // 1. no path is passed
+      expect(function () {
```

Comment:
unless there's a function context involved, this can be simplified to
```js
expect(options.read).to.throwError();
```

Reply:
Good to know, that's way more succinct.